connexion/middleware/abstract.py

```diff
@@ -1,42 +1,185 @@
 import abc
 import logging
 import pathlib
+import sys
 import typing as t
 
-import typing_extensions as te
 from starlette.types import ASGIApp, Receive, Scope, Send
 
-from connexion.apis.abstract import AbstractSpecAPI
-from connexion.exceptions import MissingMiddleware
+from connexion.exceptions import MissingMiddleware, ResolverError
 from connexion.http_facts import METHODS
+from connexion.jsonifier import Jsonifier
 from connexion.operations import AbstractOperation
-from connexion.resolver import ResolverError
+from connexion.resolver import Resolver
+from connexion.spec import Specification
 
-logger = logging.getLogger("connexion.middleware.abstract")
+logger = logging.getLogger(__name__)
```

Comment:
small nit: There is a mix of `__name__` and strings across files, also in this PR.

Reply:
Yes, I think `__name__` is a better choice than hardcoding. So I'm changing it when I touch the code, but haven't applied this change in a structured way.